oct/tests/deployment/deploy_app.py

```diff
@@ -0,0 +1,48 @@
+# pylint: disable=no-self-use # pyATS-related exclusion
+# pylint: disable=no-member
+# pylint: disable=invalid-name
+# pylint: disable=too-many-branches
+import os
+from pyats.topology import Device, loader
+from pyats.aetest import Testcase, test
+from pyats.utils.fileutils import FileUtils
+from requests.exceptions import ConnectTimeout
+from oct.tests import run_testcase
+
+
+def copy_file_to_server(device: Device) -> None:
+    futils = FileUtils(testbed=device.testbed)
+    futils.copyfile(
+        source=f"{os.path.join(os.getcwd(), 'docker-compose.yaml')}",
+        destination=f"scp://{device.testbed.devices.vm.connections.main.ip}/home/adminaccount/oct",
```

Comment:
@GoshkaEgoshka simplify configuration
```suggestion
        destination=f"scp://{device.connections.main.ip}/home/adminaccount/oct",
```